File: python/cli/prelude_cli/templates/template.cs
/*
NAME: $NAME
QUESTION: $QUESTION
CREATED: $CREATED
*/
using System;

class TTP {
    static void Test() {
        Console.WriteLine("Run test");
        Environment.Exit(100);
    }

    static void Clean() {
        Console.WriteLine("Clean up");
        Environment.Exit(100);
    }

    static void Main(string[] args) {
        if (args.Length == 0) {
            Test();
        } else {
            Clean();
        }
    }
}

File: python/cli/prelude_cli/templates/template.swift
/*
NAME: $NAME
QUESTION: $QUESTION
CREATED: $CREATED
*/
import Foundation

func test() {
    print("Run test")
    exit(100)
}

func clean() {
    print("Clean up")
    exit(100)
}

if CommandLine.arguments.contains("clean") {
    clean()
} else {
    test()
}
